src/main/java/com/opencredo/connect/venafi/tpp/log/model/EventLog.java

```diff
@@ -64,6 +65,26 @@
     private Integer Value1;
     private Integer Value2;
 
+    @VisibleForTesting
```

Comment:
Do you need this? The constructor is already package level which should be enough

Reply:
I had to add the constructor just for testing, hence the annotation. "Need" is a big word in that context. I think having it makes sense, and as I mentioned above, I expect us to use ImmutableMap going forward. It would indeed be a bit much to include all of Guava just for the purpose of using a single annotation. :)

Reply:
Took it out again.